tokens/src/components/Toast.json

```diff
@@ -1,35 +1,17 @@
 {
-  "toast": {
-    "max-width": { "value": "400px", "source": "$toast-max-width" },
-    "padding": {
-      "x": { "value": ".75rem", "source": "$toast-padding-x" },
-      "y": { "value": ".25rem", "source": "$toast-padding-y" }
-    },
-    "font": {
-      "size": { "value": ".875rem", "source": "$toast-font-size" }
-    },
-    "color": {
+  "color": {
+    "toast": {
       "base": { "value": "null", "source": "$toast-color" },
-      "background": { "value": "{color.gray.700.value}", "source": "$toast-background-color" }
-    },
-    "border": {
-      "width": { "value": "1px", "source": "$toast-border-width" },
-      "color": { 
+      "background": { "value": "{color.gray.700.value}", "source": "$toast-background-color" },
+      "border": {
         "value": "{color.black.value}",
         "source": "$toast-border-color",
         "modify": [{
           "type": "alpha",
           "amount": 0.1
         }]
       },
-      "radius": { "value": ".25rem", "source": "$toast-border-radius" }
-    },
-    "box-shadow": {
-      "value": "0 1.25rem 2.5rem rgba(0, 0, 0, .15), 0 .5rem 3rem rgba(0, 0, 0, .15)",
-      "source": "$toast-box-shadow"
-    },
-    "header": {
-      "color": {
+      "header": {
         "base": { "value": "{color.white.value}", "source": "$toast-header-color" },
```

Comment:
I think we should switch key to `text` here, because this color applies only to text I think

Reply:
added